src/stats.jl

```diff
@@ -0,0 +1,441 @@
+u = LinRange(0, 2π, 72)
+a, b = 5.0, 2.0
+function ellipse(u; a = 2, b = 3, h = 0, k = 0)
+    Point2f(h + a/2*cos(u), k + b/2*sin(u))
+end
+
+# https://github.com/dronir/Bezier.jl
```

Comment:
Can the `Bezier.jl` code be put in it's own file? `include("bezier")`

Reply:
Yes